.github/workflows/release.yml

```
@@ -0,0 +1,254 @@
+#
+# .github/workflows/release.yml
+#
+---
+name: release
+defaults:
+  run:
+    # Prefixes all `run` commands with the following command to force them to run outside Rosetta.
+    shell: arch -arm64 bash --noprofile --norc -eo pipefail {0}
+on:
+  release:
+    types: [published]
+jobs:
+  start:
+    runs-on: macos-15
+    outputs:
+      dry_run: ${{ steps.dry_run.outputs.dry_run }}
+      mas_version: ${{ steps.mas_version.outputs.mas_version }}
+      pre_release: ${{ steps.pre_release.outputs.pre_release }}
+      release_branch: ${{ steps.release_branch.outputs.release_branch }}
+    steps:
+      # Logs event details and sets `DRY_RUN` env var
+      # https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable
+      # https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=published#release
+      - name: 🔈 Log release event
+        # This step will only run for a real 'release' event.
+        if: ${{ github.event.release.name != '' }}
+        run: |
+          echo "Triggered by a release publication event (wet run)"
+          echo "release.name: ${{ github.event.release.name }}"
+          echo "release.tag_name: ${{ github.event.release.tag_name }}"
+          echo "release.target_commitish: ${{ github.event.release.target_commitish }}"
+          echo -e "release.body: \n${{ github.event.release.body }}"
+
+      - id: dry_run
+        run: |
+          echo "DRY_RUN=false" >>"$GITHUB_OUTPUT"
```

Why is `$GITHUB_OUTPUT` used? the [linked documentation above](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable) uses `"${GITHUB_ENV}"`.
```suggestion
          echo "DRY_RUN=false" >>"${GITHUB_ENV}"
```